src/spreadsheets.js

```diff
@@ -407,7 +407,10 @@ function getRows(spreadsheetConfig, options, sheetPredicate, callback) {
                     });
                     var recordRows = [];
                     rows.slice(2).forEach(function(row) {
-                        recordRows.push(_.object(_.zip(headerRow, row)));
+                        var asObject = _.transform(_.zip(headerRow, row), function(result, pair) {
```

Comment:
I have to learn about lodash. I love the work it does. I would have coded all this manually.